Formula/zpaq.rb

```diff
@@ -4,6 +4,7 @@ class Zpaq < Formula
   url "http://mattmahoney.net/dc/zpaq715.zip"
   version "7.15"
   sha256 "e85ec2529eb0ba22ceaeabd461e55357ef099b80f61c14f377b429ea3d49d418"
+  license "Unlicense"
```

Comment:
```suggestion
  license :public_domain
```

cc @Rylan12

Reply:
I picked "Unlicense" based on this file `COPYING` which includes:
```
For more information, please refer to <http://unlicense.org>
```

Reply:
I'm getting mixed messages from the readme/homepage/`COPYING`. First, note that `:public_domain` is not the same as [`Unlicense`](https://spdx.org/licenses/Unlicense.html) (which is a valid open-source license).

From the [homepage](http://mattmahoney.net/dc/zpaq.html):

> zpaq is written by Matt Mahoney and released to the public domain. It includes code from libdivsufsort 2.0 (C) Yuta Mori, 2003-2008, MIT license, public domain code for AES from libtomcrypt by Tom St Denis and public domain code for salsa20 by D. J. Bernstein

This indicates `all_of: [:public_domain, "MIT"]`

From the `readme`:

> Code from divsufsort.c is embedded in libzpaq.cpp.
> divsufsort.c is (C) 2003-2008 Yuta Mori, MIT license (see source code).
> It is also available from libdivsufsort-lite 2.0 from
> http://code.google.com/p/libdivsufsort/
> All remaining code is public domain. See COPYING.

This indicates `all_of: [:public_domain, "MIT"]`. However, the `readme` also says:

> ```
> File            Ver.   Description
> -----------     ----   -----------
> COPYING                Unlicense.
> ```

The `COPYING` file contains the [Unlicense license text](https://spdx.org/licenses/Unlicense.html) and the [MIT license text](https://spdx.org/licenses/MIT.html).

Note that while the `Unlicense` text includes the line `This is free and unencumbered software released into the public domain.`, it does not mean that `:public_domain` is accurate.

---

So it seems that the correct license is either:

```ruby
license all_of: [:public_domain, "MIT"]
```

or

```ruby
license all_of: ["Unlicense", "MIT"]
```